manifests/f/FastCopy/FastCopy/3.92/FastCopy.FastCopy.yaml

```
@@ -1,7 +1,7 @@
 PackageIdentifier: FastCopy.FastCopy
 PackageVersion: 3.92
 PackageName: FastCopy
-Publisher: FastCopy
+Publisher: H.Shirouzu & FastCopy Lab, LLC
```

```suggestion
Publisher: H.Shirouzu
```
Windows Package Manager requires the `Publisher` key in the manifest to match whatever's shown in Control Panel for the best application upgrade experience.

---

I see. Thank you.